pages/guide.py

```diff
@@ -554,6 +554,8 @@ def process_post_data(self, feature_id, stage_id=0):
 
     if self.touched('standardization'):
       feature.standardization = int(self.form.get('standardization'))
+    if self.form.get('accurate_as_of'):
+      feature.accurate_as_of = datetime.now()
```

Comment:
@jrobbins There are no other ways currently to update the `accurate_as_of` field outside of the new page that is only linked in notification emails. Hopefully, this is okay.

Reply:
That's fine for now.